cypress/e2e/checkout.cy.js

```diff
@@ -4,14 +4,16 @@
 import CartPage from '../support/pages/cart.pageObject';
 import HomeAndCataloguePageObject from '../support/pages/homeСatalogue.pageObject';
 import ContactFormPageObject from '../support/pages/contactForm.pageObject';
+import PageObject from '../support/PageObject';
```

Comment:
no need to add it over here since you are importing it at the very beginning of each pageObject.js file, you had just to add pageObject.js file to commit